src/olympia/reviewers/tests/test_code_manager_tags.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from django.test.utils import override_settings
+
+from olympia.reviewers.templatetags import code_manager_tags
+from olympia.amo.tests import TestCase
+
+
+class TestCodeManagerUrl(TestCase):
+
+    def setUp(self):
+        self.cm_url = 'http://code-manager'
+
+    def test_create_a_browse_url(self):
+        addon_id = 1
```

Comment:
You probably want to add those in the `setUp` method and use `self.addon_id`, etc.